src/services/file/interfaces/configuration.ts

```diff
@@ -12,4 +12,5 @@ export interface S3FileConfiguration {
 
 export interface LocalFileConfiguration {
   storageRootPath: string;
+  localFilesHost?: string;
```

Comment:
Add comments about what's the difference between these 2, in which case they are used, etc.